crates/epoch-snark/src/epoch_block.rs

```diff
@@ -85,8 +85,8 @@ pub fn hash_first_last_epoch_block(
     first: &EpochBlock,
     last: &EpochBlock,
 ) -> Result<Vec<bool>, EncodingError> {
-    let h1 = first.blake2()?;
-    let h2 = last.blake2()?;
+    let h1 = first.blake2(false)?;
+    let h2 = last.blake2(true)?;
```

Comment:
Isn't this a bit confusing? Why can we hash the aggregated pubkey in the last header, but not in the first one?

Reply:
Will replace with explicit function call